change-notes/1.22/analysis-java.md

```diff
@@ -5,6 +5,10 @@
 | **Query**                  | **Expected impact**    | **Change**                                                       |
 |----------------------------|------------------------|------------------------------------------------------------------|
 | Equals method does not inspect argument type (`java/unchecked-cast-in-equals`) | Fewer false positive and more true positive results | Precision has been improved by doing a bit of inter-procedural analysis and relying less on ad-hoc method names. |
+| Local-user-controlled data in arithmetic expression (`java/tainted-arithmetic-local`) | Fewer false positive results | Precision has been improved in several ways, in particular, by better detection of guards along the data-flow path. |
+| Uncontrolled data in arithmetic expression (`java/uncontrolled-arithmetic`) | Fewer false positive results | Precision has been improved in several ways, in particular, by better detection of guards along the data-flow path. |
+| Use of extreme values in arithmetic expression (`java/extreme-value-arithmetic`) | Fewer false positive results | Precision has been improved in several ways, in particular, by better detection of guards along the data-flow path. |
```

Comment:
The queries `java/tainted-arithmetic-local` and `java/extreme-value-arithmetic` are not included in the LGTM suite, so it might be confusing to include a release note for them in this file (it could optionally be moved to the QL-only release note file instead, but I think historically we have just omitted release notes for queries not included in LGTM).